srcpkgs/musl-legacy-compat/files/error.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _ERROR_H_
+#define _ERROR_H_
+
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+
+#warning usage of non-standard #include <error.h> is deprecated
+
+static unsigned int error_message_count = 0;
```

Comment:
This `error_message_count` is internal linkage, thus isn't shared by different translation units.
It may work when removing static and add `-fcommon` but `-fcommon` hurts compiler optimization.

Reply:
I'm not sure any of the applications actually use it... Maybe we can just remove it.

Reply:
Ok, it's definitely used by elfutils already... For what it's worth, this is the file that was already there and I didn't observe any issues... Or at least I think I didn't. Most of the usage happens inside individual applications.

recode doesn't use the variable either.

And it turns out kbd doesn't require it at all.

Reply:
I'm going to leave it as static for the moment.